packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -2917,14 +2990,17 @@ class _InputDateRangePickerState extends State<_InputDateRangePicker> {
   Widget build(BuildContext context) {
     final MaterialLocalizations localizations = MaterialLocalizations.of(context);
     final InputDecorationTheme inputTheme = Theme.of(context).inputDecorationTheme;
```

Comment:
Add `final ThemeData theme = Theme.of(context);` and use `theme` here and on the next line.